cvat/apps/dataset_manager/formats/README.md

```diff
@@ -536,8 +526,8 @@ archive.zip/
   - `score` (number) - the annotation `score` field
   - arbitrary attributes - will be stored in the `attributes` annotation section
 
-
-*Note*: there is also a [support for COCO keypoints over Datumaro](https://github.com/openvinotoolkit/cvat/issues/2910#issuecomment-726077582)
+Support for COCO tasks via Datumaro is described [here](https://github.com/openvinotoolkit/datumaro/blob/develop/docs/formats/coco_user_manual.md#export-to-coco)
+For example, export COCO keypoints:
```

Comment:
The link included useful information about logic of this export, I think, it should be either described here, or the link should be kept.

Reply:
the link is returned